kedro/templates/project/{{ cookiecutter.repo_name }}/setup.cfg

```
@@ -1,3 +1,7 @@
+[flake8]
+max-line-length = 88
+extend-ignore = E203
```

```suggestion
max-line-length=88
extend-ignore=E203
```
Copied from https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length, style and all. Would you prefer I remove the surrounding spaces to be consistent with the `isort` section below?